### ISSUE TITLE
Showcase core features during install

### ISSUE THREAD

**Issue by BWPanda** [backdrop/backdrop-issues#4138]
Similar to how Linux shows you some of its features when you're installing it (see screenshots below), I wonder if we can do something similar with Backdrop. With issues like https://github.com/backdrop/backdrop-issues/issues/467 the UI installer may end up being mostly a progress bar as things are setup and prepared, so let's showcase some of Backdrop's feature while the user waits for this...

![Ubuntu install](https://tutorials.ubuntu.com/es6-bundled/src/codelabs/tutorial-install-ubuntu-desktop/img/127852204903775.png)

![Linux Mint install](https://itsfoss.com/wp-content/uploads/2013/12/Linux-Mint-Installation-14.jpeg)

**Comment by domaingood** [comment]
Yes! really good UI thanks @BWPanda

**Comment by indigoxela** [comment]
That's a lovely idea, but how many images could be shown within those 10 to 15 seconds it needs to write the db tables an config. ;)

And after the progress bar reached 100%, there's some more to do: setting site name, create admin user, set timezone.

**Comment by stpaultim** [comment]
I know we discussed something like this for Tugboat demo sites. We talked about showing a short video or information while demo sites were being spun up, except that they spin up much too quickly for this to be useful. I am surprised that this issue has not come up before for the normal install process (or has it @klonos?).

It takes over 5 minutes to run through install process on my local machine, using Lando, with my custom install profile. I will look to see how long this takes on the server.

**Comment by klonos** [comment]
It takes me only 5-10sec to run the installer on my local (same as how long it takes for the tugboat demo to spin up), so I guess that's why I never revisited this idea. This time is simply not enough to do something useful.

...perhaps there is more merit to add something like that to our fancy dashboard? ...a new "Tutorial" block perhaps?

Having said all that, perhaps we can have some sort of timer going on, and if we detect that the installation is happening on a slow environment, then show some sort of slideshow?

**Comment by domaingood** [comment]
@klonos I agree with that Better one click installation more simple with good UI that,s all.I think we are not install huge data. Just need good UI with one click installation.

**Comment by stpaultim** [comment]
So, I just testing my install profile on our Linode server and the install process was less than 10 seconds. The same install profile takes 5 minutes in my local Lando environment. 

Anyway, unless your installing on my laptop, that's not much time to showcase features. :-)

**Comment by BWPanda** [comment]
Fair enough.

**Comment by klonos** [comment]
We could use the free white space in the installer for a call to action perhaps... like "please consider donating", or "attend an upcoming Drupal/Backdrop event", or "join the community Gitter/forum" etc. 5-10sec would be enough for such things (which would open in a new browser tab if clicked).

**Comment by stpaultim** [comment]
@BWPanda - I like this line of thought a lot. We should look for opportunities like this and I never meant to suggest that even this shouldn't be done, only that it's impact might be limited. Are there better ways to accomplish this goal?

Maybe a "tip of the day" or "feature of the day" block on the Dashboard?

**Comment by klonos** [comment]
> Maybe a "tip of the day" or "feature of the day" block on the Dashboard?

Yeah, I am constantly taking screenshots of how other products are doing great things UX-wise as well as marketing/outreach-wise, in the hope that I will some day create an issue for something similar for us. This one is a recent one:

<img width="968" alt="Screen Shot 2019-10-03 at 8 08 31 pm" src="https://user-images.githubusercontent.com/2423362/66963532-9faf5b00-f0bf-11e9-9c76-706f3d7c7e6a.png">

**Comment by laryn** [comment]
Maybe not enough time for a slideshow, but surely for a single slide (if we could decide what to feature).
